api/config/routes.rb

```diff
@@ -91,14 +91,7 @@
     resources :states, only: [:index, :show]
 
     resources :taxonomies do
-      member do
-        get :jstree
-      end
-      resources :taxons do
-        member do
-          get :jstree
```

Comment:
We should probably drop this into an extension in case anyone has an iOS or Android app relying on these endpoints.